packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -23,7 +24,8 @@ export function createKusama (t: TFunction): EndpointOption {
       Parity: 'wss://kusama-rpc.polkadot.io',
       OnFinality: 'wss://kusama.api.onfinality.io/public-ws',
       'Patract Elara': 'wss://kusama.elara.patract.io',
-      Pinknode: 'wss://rpc.pinknode.io/kusama/explorer'
+      Pinknode: 'wss://rpc.pinknode.io/kusama/explorer',
+      'light client': createProviderUrl('kusama-substrate-connect', 'substrate-connect')
```

Comment:
Now that I see this, we may have a small issue now. So the provider URLs are actually selected randomly (based on feedback, i.e. it spreads the loads between the providers). So this may now be randomly selected and based on what the user has available, may not work.

... will continue reading through, this may be addressed later on.

Reply:
Indeed, this will be problematic - https://github.com/polkadot-js/apps/blob/master/packages/apps/src/Endpoints/Network.tsx#L28

For the time being I am guessing we need to filter. Or, if they have light-client capabilities, select from the light-client list only. And it not (or no light clients available), select from the other "normal" list.

Reply:
> For the time being I am guessing we need to filter. Or, if they have light-client capabilities, select from the light-client list only. And it not (or no light clients available), select from the other "normal" list.

I would remove the light-clients for now (as it is experimental and user should know that chooses it).
It should always be randomly selected from the "normal" list only - and user can manually switch to light client if he/she wishes to do so